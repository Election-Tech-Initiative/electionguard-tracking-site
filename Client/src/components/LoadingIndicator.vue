<template>
  <div class="loading-indicator control is-loading"></div>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

@Component
export default class LoadingIndicator extends Vue {
}
</script>

<style lang="scss" scoped>
.loading-indicator.control.is-loading {
  display: flex;
  justify-content: center;
  height: 4rem;
  &::after {
    font-size: 3.5rem;
    left: auto;
    right: auto;
    top: 0;
    border-width: 0.2rem;
  }
}
</style>
