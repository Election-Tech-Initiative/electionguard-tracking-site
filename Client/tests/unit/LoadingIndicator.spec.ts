import { mount } from '@vue/test-utils';

import LoadingIndicator from '@/components/LoadingIndicator.vue';

describe('LoadingIndicator.vue', () => {
    it('renders', () => {
        const wrapper = mount(LoadingIndicator);
        expect(wrapper.element).toMatchSnapshot();
    });
});
