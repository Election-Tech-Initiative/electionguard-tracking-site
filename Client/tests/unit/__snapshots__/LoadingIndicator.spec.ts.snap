// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator.vue renders 1`] = `
<div
  class="loading-indicator control is-loading"
/>
`;
